src/lib/tokenStrategy.js

```diff
@@ -10,10 +10,10 @@ class TokenStrategy extends passport.Strategy {
 
   authenticate (req) {
     const token = req.query.token
```

Comment:
As discussed, this should be switched from putting the token in the query string to adding a bearer token auth header